Resources/Prototypes/Floof/Entities/Mobs/Cyborgs/quadborg.yml

```diff
@@ -37,4 +44,17 @@
   - type: Speech
     speechVerb: Robotic
   - type: LayingDown
-  - type: LeashAnchor # Floofstation
\ No newline at end of file
+  - type: LeashAnchor # Floofstation
+  
+- type: inventoryTemplate
+  id: Securityborg
+  slots:
+  - name: pocket1
+    slotTexture: pocket
+    slotFlags: POCKET
+    slotGroup: MainHotbar
+    stripTime: 3
+    uiWindowPos: 0,3
+    strippingWindowPos: 0,4
+    displayName: Pocket 1
+    stripHidden: false
```

Comment:
```suggestion
```

Reply:
We not allowing to provide borgs with pockets/inventory.

Reply:
the pocket is required to allow the borg to be able to apprehend with cuffs

Reply:
umm when I made a borg you didn't need any of that just need to put the cuff in the mod and it was useable

Reply:
the cuffs act like they can be used but they will not actually cuff people if you just stick them in the module